README.md

```diff
@@ -84,13 +84,41 @@ Following environment variables are mandatory for any successful build:
     - We use Google Cloud SDK service account authentication method [with key file](https://cloud.google.com/sdk/gcloud/reference/auth/activate-service-account) method to get access to the GKE cluster. 
     - The generated key file should be included into the project but in encrypted form. Travis CI supports [file encryption](https://docs.travis-ci.com/user/encrypting-files/) that greatly facilitates and simplify the solution.
     - Please refer to this example `travis encrypt-file super_secret.txt --add` to understand how the encryption works and how Travis populates these two environment variable.
-- **GKE_PROJECT** - the name of target Google Cloud Project.
-- **GKE_ZONE** - the name of the target GCP cluster zone.
-- **GKE_CLUSTER** - the name of the target GCP cluster. Note that the cluster must be precreated. See "_1.2. Deploy to Google Kubernetes Engine (GKE)_"
-- **GKE_NAMESPACE** - the target Kubernetes namespace. 
+- **CLOUDSDK_CORE_PROJECT** - the name of target Google Cloud Project.
```

Comment:
don't you need a doc entry for DOCKER_IMAGE too?

Reply:
I want to calculate this using [Github API](https://developer.github.com/v3/)

Reply:
Added note about `DOCKER_IMAGE` environment variable into [this section](https://github.com/andrewkandzuba/openexchange-webservice-go/tree/google-cloud-build#12-deploy-to-google-kubernetes-engine-gke)